outputs/yandex.go

```diff
@@ -0,0 +1,77 @@
+package outputs
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+
+	"github.com/falcosecurity/falcosidekick/types"
+)
+
+// NewAWSClient returns a new output.Client for accessing the AWS API.
+func NewYandexS3Client(config *types.Configuration, stats *types.Statistics, promStats *types.PromStatistics, statsdClient, dogstatsdClient *statsd.Client) (*Client, error) {
+
+	if config.Yandex.AccessKeyID != "" && config.Yandex.SecretAccessKey != "" {
+		err1 := os.Setenv("AWS_ACCESS_KEY_ID", config.Yandex.AccessKeyID)
+		err2 := os.Setenv("AWS_SECRET_ACCESS_KEY", config.Yandex.SecretAccessKey)
```

Comment:
if both AWS and Yandex are used, there will be a conflict. Is Yandex also able to use different keys?